javascript/javascript_sdk/package.json

```diff
@@ -11,10 +11,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "google-protobuf": "^3.12.0",
+    "google-protobuf": "3.14.0",
     "mathjs": "^7.0.0",
     "yargs": "^15.3.1",
-    "grpc-web": "^0.4.0"
+    "grpc-web": "^0.4.0",
+    "websocket": "1.0.33",
+    "ws": "7.4.2"
```

Comment:
Just curious, what's the difference between `websocket` and `ws`?

Reply:
ws: WebSocket server and client for node.js. One of the fastest libraries if not the fastest one.

websocket-node: WebSocket server and client for node.js

I believe they have almost identical functionality but ws is slightly faster. Also way to import websocket is different from ws. you can find more about these in https://www.npmjs.com/package/ws, and https://www.npmjs.com/package/websocket. I will remove one websocket for now since I am not using them

Reply:
Alright, sounds good. Thanks for the info